### PR TITLE
For #11201: Updates 'synced tabs' and account settings 'tabs' strings

### PR DESCRIPTION
**Pull request by ValentinTimisica**
### Pull Request checklist

- [x] **Tests**: This PR does not include thorough tests. Strings changes.
- [x] **Screenshots**: This PR does not include screenshots or GIFs of the changes made.
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md).

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
     <!-- Browser menu toggle that installs a Progressive Web App shortcut to the site on the device home screen. -->
     <string name="browser_menu_install_on_homescreen">Install</string>
     <!-- Menu option on the toolbar that takes you to synced tabs page-->
-    <string name="synced_tabs">Synced Tabs</string>
+    <string name="synced_tabs_2">Synced tabs</string>
     <!-- Browser menu button that opens the find in page menu -->
```
Comment (cadeyrn) [inline app/src/main/res/values/strings.xml:100]: Please see https://github.com/mozilla-mobile/fenix/commit/44d9dc9237e054349da832487a3d94384a2663e8#r39455402

> changing capitalization to existing strings doesn't require to use new IDs. The reason is that, by changing the ID, we require all languages to retranslate. But each language has its own capitalization rules, so they should already be using them consistently, independently from what English does.

Comment (BranescuMihai) [reply app/src/main/res/values/strings.xml:100]: @cadeyrn thanks for the reminder! 
@flodolo if we don't change the ID, so we don't retranslate the strings, there will be languages (for example English CA) that will still keep the wrong capitalisation. So is there a way to say: hey, there are multiple languages that don't respect the correct capitalisation rule, please look over them, without changing the ID?

Comment (flodolo) [reply app/src/main/res/values/strings.xml:100]: There's no way, but you can ping me and I can fix the English variants directly.

Comment (BranescuMihai) [reply app/src/main/res/values/strings.xml:100]: got it, thanks!

**Comment by ValentinTimisica** [comment]
@flodolo I changed the strings only for one English variant. From what I understand, after this is merged you will change the other english variants?

**Comment by flodolo** [comment]
> @flodolo I changed the strings only for one English variant. From what I understand, after this is merged you will change the other english variants?

I'll change them right away, so I won't risk forgetting about it.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -289,7 +289,7 @@
     <!-- Preference for syncing logins -->
     <string name="preferences_sync_logins">Logins</string>
     <!-- Preference for syncing tabs -->
-    <string name="preferences_sync_tabs">Tabs</string>
+    <string name="preferences_sync_tabs">Open tabs</string>
```
Comment (flodolo) [inline app/src/main/res/values/strings.xml:292]: Wait, this one is not OK. You're not just changing capitalization, you need a new ID for this.

Comment (ValentinTimisica) [reply app/src/main/res/values/strings.xml:292]: Oh right, my bad. I'll change it now.

Comment (ValentinTimisica) [reply app/src/main/res/values/strings.xml]: Done

**Pull request merged**